src/renderer/widgets/RemoveVoteModal/ui/RemoveVoteModal.tsx

```diff
@@ -1,35 +1,59 @@
 import { type ApiPromise } from '@polkadot/api';
 import { useGate, useUnit } from 'effector-react';
+import { useEffect, useState } from 'react';
 
 import { useI18n } from '@app/providers';
-import { type AccountVote, type Asset, type Chain, type OngoingReferendum } from '@shared/core';
-import { useModalClose } from '@shared/lib/hooks';
-import { Step, isStep } from '@shared/lib/utils';
+import { type AccountVote, type Address, type Asset, type Chain, type OngoingReferendum } from '@/shared/core';
+import { useModalClose } from '@/shared/lib/hooks';
+import { Step, isStep, nonNullable, nullable } from '@/shared/lib/utils';
 import { BaseModal, Button } from '@shared/ui';
-import { OperationTitle } from '@entities/chain';
-import { OperationResult } from '@entities/transaction';
-import { OperationSign, OperationSubmit } from '@features/operations';
-import { RemoveVoteConfirmation, basketUtils } from '@features/operations/OperationsConfirm';
+import { OperationTitle } from '@/entities/chain';
+import { OperationResult } from '@/entities/transaction';
+import { OperationSign, OperationSubmit } from '@/features/operations';
+import { RemoveVoteConfirmation, basketUtils } from '@/features/operations/OperationsConfirm';
+import { SignatorySelectModal } from '@/pages/Operations/components/modals/SignatorySelectModal';
```

Comment:
if we use `SignatorySelectModal` here from `pages/Operations` maybe it's better to move this component into feature?

Reply:
I think it will be great refactoring later :)